courses/machine_learning/deepdive/09_sequence/labs/sinewaves.ipynb

```diff
@@ -4,259 +4,833 @@
    "cell_type": "markdown",
```

Comment:
<p>This is a very nice write-up.  My only comment is to make "keras": "Keras".  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1001/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>